module/data/item/spell.mjs

```diff
@@ -212,6 +242,8 @@ export default class SpellData extends ItemDataModel.mixin(ActivitiesTemplate, I
     }, { all: [], vsm: [], tags: [] });
     labels.components.vsm = game.i18n.getListFormatter({ style: "narrow" }).format(labels.components.vsm);
 
+    this.preparation.static = (CONFIG.DND5E.spellcasting[this.preparation.mode] ?? { static: true }).static === true;
```

Comment:
A spell belongs to a "static" preparation mode if said prep mode does not exist, or is `static: true`.